.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: K3Pomdav7dFeKalQEyIcFu1R9hoP007gg
```

Comment:
check this out @andela-aadepoju 
https://circleci.com/gh/andela-iamadi/zhishi-backend/edit#env-vars